ext/bcmath/libbcmath/src/recmul.c

```diff
@@ -220,6 +220,34 @@ static void bc_write_bcd_representation(uint32_t value, char *str)
 	memcpy(str, &digits, sizeof(digits));
 }
 
+static inline void bc_mul_vector_to_bc_num(BC_VECTOR *prod_vector, size_t prodlen, size_t prod_arr_size, bc_num *prod)
```

Comment:
It might be useful to add a brief comment to mention that this code implements the Karatsuba algorithm (https://en.wikipedia.org/wiki/Karatsuba_algorithm). That could ease future maintenance.

We do the same in other parts of the code. For example: https://github.com/php/php-src/blob/bfc988552f6b0f2e950b2221046e7fdc5e2d595a/ext/standard/soundex.c#L21 and https://github.com/php/php-src/blob/bfc988552f6b0f2e950b2221046e7fdc5e2d595a/Zend/Optimizer/scdf.c#L22-L24

Thanks!